server/src/integrations/message-queue/drivers/local.driver.ts

```diff
@@ -0,0 +1,20 @@
+import { EventEmitter } from 'stream';
+
+import { MessageQueueDriver } from './interfaces/message-queue-driver.interface';
+
+export class LocalDriver extends EventEmitter implements MessageQueueDriver {
+  constructor() {
+    super();
+  }
+
+  async subscribe<T>(
+    topic: string,
+    handler: ({ data, id }: { data: T; id: string }) => Promise<void>,
+  ) {
+    this.on(topic, handler);
+  }
+
+  async publish<T>(topic: string, data: T): Promise<void> {
+    this.emit(topic, { data, id: Math.random() });
```

Comment:
import { v4 } from 'uuid'?

Reply:
okay i can do that, since its only for local dev, i dint do it.